client_test.go

```diff
@@ -612,118 +612,3 @@ func TestNewWithLocalAddr(t *testing.T) {
 	assertNil(t, err)
 	assertEqual(t, resp.String(), "TestGet: text response")
 }
-
-func TestClientOnResponseError(t *testing.T) {
```

Comment:
Why are these two test functions removed?

Reply:
I'm sorry that I do a wrong rebased operation. I will fix it.